src/frameworks/internal/messenger/messenger.js

```diff
@@ -188,18 +188,43 @@ cwc.framework.Messenger.prototype.postMessage = function(name, value) {
  * @private
  */
 cwc.framework.Messenger.prototype.executeCode_ = function(code) {
-  if (!code || typeof code !== 'string') {
-    return;
+  let id = false;
+  switch (typeof code) {
+    case 'string':
+      break;
```

Comment:
Please move line 214  - 228 into this case so that all cases are handled inside the switch statement.

Reply:
Those lines need to be in both cases of the switch, but I can just make them a function and call it in both cases and eliminate the if (id) below.